packages/dapp/src/utils/helpers.js

```diff
@@ -61,11 +61,23 @@ export const getResolverInfo = (chainId, resolver) =>
 export const getTokens = (chainId, allTokens) =>
   allTokens[chainId] || allTokens[4];
 
-export const getTokenInfo = (chainId, token, tokenData) =>
-  (tokenData[chainId] || tokenData[4])[token] || {
-    decimals: 18,
-    symbol: 'UNKNOWN',
-  };
+export const getTokenInfo = (chainId, token, tokenData) => {
+  if (!tokenData || Object.keys(tokenData || {}).length === 0) {
+    return {
+      decimals: 18,
+      symbol: 'UNKNOWN',
+    };
+  }
+  // default chainId is 5 for goerli test network
```

Comment:
yes. I created a new constant so it can be updated in one place, and easier to find by global searching. Will need to catch this when Goerli is deprectated.